server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/CassandraBlobModule.java

```diff
@@ -19,35 +19,53 @@
 
 package org.apache.james.blob.cassandra;
 
-import static org.apache.james.blob.cassandra.BlobTables.BlobParts.CHUNK_NUMBER;
-import static org.apache.james.blob.cassandra.BlobTables.BlobParts.DATA;
-import static org.apache.james.blob.cassandra.BlobTables.BlobTable.ID;
-import static org.apache.james.blob.cassandra.BlobTables.BlobTable.NUMBER_OF_CHUNK;
-import static org.apache.james.blob.cassandra.BlobTables.BlobTable.TABLE_NAME;
+import static org.apache.james.blob.cassandra.BlobTables.DefaultBucketBlobParts.DATA;
```

Comment:
we should not use static import now that we have several similar tables, don't you think?